content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Private Cloud Gateway"
+metaTitle: "Private Cloud Gateway"
+metaDescription: "Troubleshooting steps for Private Cloud Gateway."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Private Cloud Gateway (PCG)
+
+The following are scenarios you may encounter when deploying and using a Private Cloud Gateway.
+
+<br />
+
+
+# Scenairo - Jet Crashback Loop
+
+During the installation of a PCG cluster, one of the internal Palette components will continue to undergo a CrashLoopBackOff until the PCG is configured correctly in the User Interface (UI). The internal component, *Jet*, will transition to a healthy state once the PCG cluster is successfully registered with Palette.
+
+## Debug Steps
+
+<br />
+
+Complete the PCG installation so that the internal component receives the proper authorization token from Palette and completes the initialization successfully.
+
+<br />
+
+# Scenario - PCG Installer Unable to Register With Palette
+
+When the PCG installer instance is powered on, the PCG installer goes through a bootstrap process and attempts to register itself with Palette. This process typically takes between five to ten minutes. If the installer fails to register with the Palette within the timeframe, it could indicate a bootstrapping error.
+
+## Debug Steps
+
+<br />
+
+1. SSH into the PCG installer virtual machine using the username `ubuntu` and the SSH key provided during OVA import.
+
+
+2. Inspect the log file located at **/var/log/cloud-init-output.log**. 
+
+  <br />
+
+  ```bash
+  cat /var/log/cloud-init-output.log
+  ```
+
+  This log file will contain error messages if there are failures with connecting to Palette, authenticating, or downloading installation artifacts. A common cause for these errors is incorrect values provided to the installer, such as the Palette endpoint or a mistyped pairing code.
+
+<br />
```

Comment:
Remove the break in line 55. 

@karl-cardenas-coding I wasn't able to find a how-to to delete a line in a PR. I'll try to remember to ask you.